### ISSUE TITLE
Framer X release?

### ISSUE THREAD

**Issue by imdaveho** [koenbok/Framer#587]
Hi @koenbok is there an ETA for when framer.js is going to move to the new React/Framer X paradigm?

**Comment by dkgrieshammer** [comment]
omg please don't

**Comment by imdaveho** [comment]
@dkgrieshammer Can I ask for additional context here? Wouldn't it make sense to converge the two versions? To be able to programmatically design UX that aligns or can interop with the desktop client was exactly the use case of the original framer.js correct?

**Comment by dkgrieshammer** [comment]
nope, "original" framerjs didn't have an app/desktop client - it's an opensource javascript library with an optional but really nice generator app added (no coffee needed there). 
React (today's jquery) is really nice and used all over but sometimes its simple overkill for smaller prototypes and most of all it's pretty opinionated on how it should be included with other ways of organisation. So I would prefer to keep them apart or at least branch; just my five cents ;)
